pkg/deviceshifu/deviceshifusocket/deviceshifusocket.go

```diff
@@ -226,3 +194,34 @@ func (ds *DeviceShifu) Start(stopCh <-chan struct{}) error {
 func (ds *DeviceShifu) Stop() error {
 	return ds.base.Stop()
 }
+
+func praseCommandWithEncode(input string, encode string) ([]byte, error) {
```

Comment:
are we decoding the command in this function? if so change the function name accordingly